docs/guide/connected-apps.md

```diff
@@ -0,0 +1,31 @@
+title: Connected Apps Guide
+description: Connected Apps  is a data pivoting feature and available in all solutions. You can connect any two Apps you have access to, regardless of their type.
+
+Connected Apps  is a data pivoting feature and available in all solutions. You can connect any two Apps you have access to, regardless of their type.
+
+Connecting a Logs App to a Monitoring App will **pre-select that Logs App when you decide to correlate metrics from the connected Monitoring App.**
+
+To connect Apps, click on the connect Apps icon on top right from any report and click **“+ Connect New App”** button 
+
+![Sematext Cloud Connected Apps - Connect Apps Drop Down](../images/guide/connected-apps/connect-apps-dropdown.png)
+
+Select the Apps and click **“Connect Apps”** button
+
+![Sematext Cloud Connected Apps - Connected Apps Page](../images/guide/connected-apps/connect-apps-page.png)
+
+Once you connect Apps together, you can easily navigate from one to another with a single click or open the second App in [Split Screen](https://sematext.com/docs/guide/split-screen/) to **correlate metrics and logs**.
```

Comment:
s/together//
from one to another => from one App to the other 

> to **correlate metrics and logs**.

Not just that, though. e.g. you can correlate 2 Monitoring Apps or Experience and Monitoring or Logs and Experience.  Not sure about Synthetics.

Reply:
Yes I am just not sure how to state this. 
Maybe like this?
Connecting Apps will pre-select the second App when you want to correlate metrics, logs, monitor run results or data from real user sessions.

Reply:
Simply:
... to correlate their data.